### PR TITLE
Added DDD Hack Day Blog

### PR DESCRIPTION
**Pull request by tjforryan**

### CODE REVIEW

File: internal/_posts/2016-01-11-the-daily-developer-dashboard.markdown [hunk 1]
```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+title:  The DDD Hack Day
+date:   2016-01-11 11:22:30
+author: Tom Forryan
+categories: blog category
```
Comment (craggy2593) [inline internal/_posts/2016-01-11-the-daily-developer-dashboard.markdown]: You might as well just remove the category from this yaml

**Comment by craggy2593** [comment]
Looks good. Content seems fine! You might want to add some more of the images possibly, but thats down to you.

**Comment by craggy2593** [comment]
Looks good! :shipit:

**Comment by tcrands** [comment]
Looks good! Going to merge

**Pull request merged**